host/sgx/windows/sgxquoteexloader.c

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#if defined(OE_LINK_SGX_DCAP_QL)
+
+#if __has_include(<sgx_uae_quote_ex.h>) && __has_include(<sgx_urts.h>)
+#define OE_LINK_SGX_QUOTE_EX
```

Comment:
I would discourage any additional build time dependency variant behavior since we want to minimize the variety of libraries that OE SDK has to carry/release. Be aware that the general direction is to move away from these kinds of build flags, see #3270 which also interacts with this PR in its use of `OE_LINK_SGX_DCAP_QL`

Reply:
__has_include seems to be a C++17 feature. Does this work in MSVC?

Reply:
Will fix. See [another comment](https://github.com/openenclave/openenclave/pull/3268/files/9010a324028d27c02fa0bab2361b695346842917#r455991059)